### ISSUE TITLE
option to remove lookup component from action panel

### ISSUE THREAD

**Issue by balmas** [alpheios-project/components#608]
requested by @abrasax 

After #607 is implemented, in the Alpheios reader a lookup icon will open the lookup component, and in this configuration the lookup component should be no longer present in the action panel when it is opened from the tools icon

However, because non-alpheios users of the embedded library may include the floating toolbar icon instead of custom elements for these features, we still need the ability for the action panel to contain both the lookup component and the tools in that configuration.

**Comment by kirlat** [comment]
Do we need to support an ability for both the lookup and the toolbar's action panel be open at the same time? Action panel right now is a single instance object. If we need to open both the lookup and the toolbar, we'll have to have two instances of an action panel object: one for the lookup and another one for the toolbar. We have to introduce two sets of config object, Vuex store modules, etc., maybe even two different modules. That will be a pretty big change for the components library. 

If we, on the other hand, need to show either one or the other (i.e. the toolbar panel will close when the lookup is opened and vice versa) we can make do with the existing single instance of the action panel: when one variant (lookup or the toolbar) is opened and the other one happened to be open at the same time, we can close the one that was open, transform it to the other variant's configuration with configurational props, and reopen.

**Comment by balmas** [comment]
>Do we need to support an ability for both the lookup and the toolbar's action panel be open at the same time? 
No, I don't think so. Your approach to use a single instance sounds fine.

**Comment by kirlat** [comment]
This has been merged into master.

**Comment by balmas** [comment]
Will be tested vi alpheios-project/alpheios_nemo_ui#182
